specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzeconversations.json

```diff
@@ -1350,8 +1350,11 @@
       "properties": {
         "result": {
           "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
```

Comment:
Is this actually correct, though?

Reply:
It wasn't exactly correct. This is the original swagger
<img width="439" alt="image" src="https://github.com/Azure/azure-rest-api-specs/assets/135664462/9bef41e3-34ed-4f13-8715-90ea347afe88">
To match I used Record<never>, but this caused an error during SDK generation so I used Record<string>. I just changed this to use Record<unknown> instead, since this is a closer match and works with SDK generation (I just tested this). The only difference is the additionalProperties: true line.

Reply:
Why not just declare a `result` property of type `unknown`?